packages/ingest/src/import-lessons/graphql/client.ts

```diff
@@ -0,0 +1,15 @@
+import { GraphQLClient } from "graphql-request";
+
+if (!process.env.OAK_GRAPHQL_SECRET) {
+  throw new Error("OAK_GRAPHQL_SECRET is not set");
+}
+
+export const graphqlClient = new GraphQLClient(
+  `https://hasura.thenational.academy/v1/graphql`,
```

Comment:
At some point it would be good to put this behind an env var

Reply:
Agreed